autoload/ctrlp/funky/ft/thrift.vim

```diff
@@ -0,0 +1,13 @@
+" Language: Thrift
+" Author: timfeirg
+" License: The MIT License
+
+function! ctrlp#funky#ft#thrift#filters()
+  let filters = [
+        \ { 'pattern': '\m\C^[\t ]*struct[\t ]\+',
+        \   'formatter': [] },
+        \ { 'pattern': '\m\C^[\t ]*service[\t ]\+',
```

Comment:
It is suggested gathering  these patterns into one for performance pov, coz Funky tries to find patterns one by one.
So in this case like this is probably faster:
```vim
'\m\C^[\t ]*\(struct\|service\)[\t ]\+'
```